packages/venia-concept/src/components/ProductOptions/tile.css

```diff
@@ -1,23 +1,40 @@
 .root {
-    composes: root from '../clickable.css';
-    border: 1px solid rgb(var(--venia-text));
-    border-radius: 2px;
+    border: 1px solid black;
+    background-color: white;
+    width: 3rem;
     height: 3rem;
-    margin-left: 1rem;
-    margin-top: 1rem;
-    min-width: 3rem;
-    padding: 0 0.75rem;
+    composes: root from './option.css';
 }
 
-@media (min-width: 1024px) {
-    .root {
-        height: 2rem;
-        min-width: 2rem;
-        padding: 0;
-    }
+.check {
+    display: none;
 }
 
-.root_selected {
-    background-color: rgb(var(--venia-text));
+.disabled > span:after,
+.disabled > span:before {
```

Comment:
I see some duplication of style here. Could you compose this with the .disabled rules from `swatch.css` (or vice versa)?

Reply:
For `tile` and `swatch` there were definitely duplicates that could be inherited from `option.css` (option is the base for all of the different styles). For `miniTile`, I need to explicitly state the style because

1. The way I merge styles [here](https://github.com/magento-research/pwa-studio/pull/359/files/22259d3512ed2a2e8484aaf10e2d803a74e2d0d0#diff-2ad3e2d342d712afe5666c9ee107d565R65) forces me to include a class in `miniTile` that overrides the `disabled` option in `option.css`.  
2. As far as I can tell composition doesn't work for anything that isn't a plain old CSS class. Since I'm using pseudo elements composition wont work here. 

If you have any advice about a better way to utilize CSS modules, that'd be great. They seem pretty limiting compared to SCSS.